server/dao/MemoryOrderDao.cpp

```diff
@@ -0,0 +1,20 @@
+#include "MemoryOrderDao.hpp"
+
```

Comment:
Do we really need OrderDao?
1) Maybe in core it's better to user OrderBook instead of OrderDao
2) and/or Add missing functionality from OrderDao to OrderBook

Reply:
a